pennylane/pauli/dla/lie_closure.py

```diff
@@ -249,7 +249,13 @@ def __init__(self, generators, dtype=float, tol=None):
             ]
 
         # Get all Pauli words that are present in at least one Pauli sentence
-        all_pws = list(reduce(set.__or__, [set(ps.keys()) for ps in generators]))
+
+        all_pws = (
+            list(reduce(set.__or__, [set(ps.keys()) for ps in generators]))
+            if len(generators) != 0
+            else []
+        )
```

Comment:
Given this takes 5 lines now, one could also do this in a "vanilla" `if-else` clause, but that's a matter of taste of course :)

Reply:
✅